### PR TITLE
started.json not uploaded

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

File: sjb/config/common/test_cases/origin.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ actions:
         gsutil cp gcs/started.json "${path}/started.json"
         if [[ -n "${PULL_NUMBER:-}" ]]; then
           echo "${path}" > "${BUILD_NUMBER}.txt"
-          if [[ "${REPO_NAME}" != "origin" ]]; then
+          if [[ -n "${REPO_OWNER:-}" || "${REPO_NAME:-}" != "origin" ]]; then
             segment="${REPO_OWNER}_${REPO_NAME}/"
```
Comment (kargakis) [inline sjb/config/common/test_cases/origin.yml]: You can still break here if one of the parameters is unset.

Comment (smarterclayton) [reply sjb/config/common/test_cases/origin.yml]: there's no case where that is true

Comment (smarterclayton) [reply sjb/config/common/test_cases/origin.yml]: we don't accept one or the other.

Comment (stevekuznetsov) [reply sjb/config/common/test_cases/origin.yml]: This is wrong -- it will set whenever `$REPO_OWNER` is set, which should be always

**Comment by smarterclayton** [comment]
Fixed.

**Pull request merged**